s/squawk.py

```diff
@@ -68,17 +68,25 @@ def main() -> None:
     os.environ.setdefault("DEBUG", "1")
     os.environ.setdefault("DATABASE_URL", "postgres://postgres@127.0.0.1:5432/postgres")
 
+    subprocess.run(["./.venv/bin/python", "./backend/manage.py", "migrate"], check=True)
```

Comment:
is this necessary?

Reply:
Yeah I think in select cases it is. Otherwise the alter table command fails with some error when trying to sqlmigrate.